infra/modules/monitoring/main.tf

```diff
@@ -0,0 +1,149 @@
+# Retrieve url for external incident management tool (e.g. Pagerduty, Splunk-On-Call)
+data "aws_ssm_parameter" "this" {
+  name = "Incident-management-integration-url-${var.app_name}-${var.environment}"
+}
```

Comment:
i don't think it makes sense to have this data resource here. the data resource should probably be in infra/app/service/main.tf

Reply:
fixed